node/resolvers/profile/index.ts

```diff
@@ -1,41 +1,94 @@
-import { map, merge } from 'ramda'
-import {headers, withAuthToken} from '../headers'
+import http from 'axios'
+import { parse as parseCookie } from 'cookie'
+import {head, values, pickBy, pipe, prop, find } from 'ramda'
+import { headers, withAuthAsVTEXID } from '../headers'
 import httpResolver from '../httpResolver'
 import paths from '../paths'
 import profileResolver from './profileResolver'
+import ResolverError from '../../errors/resolverError'
 
-export const mutations = {
-  createAddress: httpResolver({
-    data: ({ fields }) => fields,
-    headers: withAuthToken(headers.profile),
-    merge: ({ id, fields }) => merge({ id }, fields),
-    method: 'PATCH',
-    url: account => paths.profile(account).address(''),
-  }),
-
-  deleteAddress: httpResolver({
-    headers: withAuthToken(headers.profile),
-    method: 'DELETE',
-    url: (account, { id }) => paths.profile(account).address(id),
-  }),
-
-  updateAddress: httpResolver({
-    data: ({ fields }) => fields,
-    headers: withAuthToken(headers.profile),
-    merge: ({ id, fields }) => merge({ id }, fields),
-    method: 'PATCH',
-    url: (account, { id }) => paths.profile(account).address(id),
-  }),
+const makeRequest = async (url, token, data?, method='GET') => http.request({
+  url, data, method, headers: { Authorization: token }
```

Comment:
Is this working? I'm surprised. The user `VtexIdclientAutCookie_{account}` has no powers to do anything in masterdata.. @igorframos how is this possible?

Maybe there is some router magic, as you're using `Authorization` instead of `Proxy-Authorization` + `VtexIdclientAutCookie`.

Anyway, we should drop that in favor of `Proxy-Authorization` + `VtexIdclientAutCookie`, using app's `authToken`.